lstchain/calib/camera/utils.py

```diff
@@ -0,0 +1,70 @@
+import tables
```

Comment:
The name of this module is too general for the code that is contains, that is instead rather specific.
I would not create at all this module and rather put the functions that it contains inside the main script or, better, tool

Reply:
> I would not create at all this module and rather put the functions that it contains inside the main script or, better, tool

but, this module is used also in notebook to show how pedestal cleaning is working.

Reply:
We should then give a more clear status to this module.  
Is it for generic functions related to DL1 calibration data? (eg. dl1_calibration_utils.py) Or is it only for the pixel threshold estimation? (e.g. pixel_threshold_estimation.py)
According to that I would change the name of the module (and eventually of the functions) so that it is more self-explaining.

Reply:
Not urgent, right? We can leave it like this for the coming release.

Reply:
I renamed `utils.py` -> `pixel_threshold_estimation.py`

Reply:
> Not urgent, right? We can leave it like this for the coming release.

@moralejo, why should we pospone to a coming release an easy improvement of new added code?

Reply:
Hi @FrancaCassol, there were no other pending changes, right?